src/mcts/params.cc

```diff
@@ -206,6 +209,7 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<IntOption>(kMiniBatchSizeId, 1, 1024) = 256;
   options->Add<IntOption>(kMaxPrefetchBatchId, 0, 1024) = 32;
   options->Add<BoolOption>(kLogitQId) = false;
+  options->Add<FloatOption>(kOneMinusEpsId, 0.0f, 1.0f) = 0.99999994f;
```

Comment:
Does a lower limit of 0 make sense? 
Also shouldn't the max here be also 0.99999994f? A cleaner approach may be to keep the upper limit as 1, and multiply by 0.99999994f when setting the constant (see later).

Reply:
I've changed the lower limit to 0.1 where it's close enough to plain Q that using LogitQ doesn't make sense.

Note that I also divide by scale (`logit(scale * Q)/scale`) now so Q behavior isn't changed near 0 when the scale is less than 1. I'll post a graph regarding this momentarily.